infra/gcp/ensure-main-project.sh

```diff
@@ -84,6 +89,45 @@ enable_api "${PROJECT}" secretmanager.googleapis.com
 color 6 "Ensuring the cluster terraform-state bucket exists"
 ensure_private_gcs_bucket "${PROJECT}" "gs://${CLUSTER_TERRAFORM_BUCKET}"
 
+
+color 6 "Ensuring all the prow buckets"
```

Comment:
```suggestion
color 6 "Ensuring all the prow buckets exist"
```
Super Nit just to keep it consistent with the rest of the language

Reply:
Thank you! :-)